ci/test-stable.sh

```diff
@@ -112,8 +112,13 @@ test-local-cluster)
 esac
 
 echo --- ci/localnet-sanity.sh
-export CARGO_TOOLCHAIN=+"$rust_stable"
 (
   set -x
-  ci/localnet-sanity.sh -x
+  CARGO_TOOLCHAIN=+"$rust_stable" ci/localnet-sanity.sh -x
+)
+
+echo --- ci/run-sanity.sh
+(
+  set -x
+  CARGO_TOOLCHAIN=+"$rust_stable" ci/run-sanity.sh -x
```

Comment:
Err, it seems `-x` is specified three times. xD. I'll fix this later!

Reply:
-x -x -x :)